vector/src/main/java/im/vector/app/features/home/room/detail/composer/RichTextComposerLayout.kt

```diff
@@ -56,12 +50,13 @@ class RichTextComposerLayout @JvmOverloads constructor(
 
     private val views: ComposerRichTextLayoutBinding
 
-    override var callback: PlainTextComposerLayout.Callback? = null
+    override var callback: Callback? = null
 
     private var currentConstraintSetId: Int = -1
-
     private val animationDuration = 100L
 
+    private var isFullScreen = false
```

Comment:
In case of configuration change, this value (if true) may be lost.
We have 2 source of truth, this is not ideal. Can we rely only on the ViewState?